### ISSUE TITLE
Idiot-proofing issue: Project delete button too close to resource delete button

### ISSUE THREAD

**Issue by JoyfulGen** [DDMAL/Rodan#1079]
Disclaimer: I did a dumb thing, so I now hope to help prevent other people from doing the dumb thing.

In the new rodan.staging UI (big fan, by the way), if you go to the `Resources` tab and select a resource, a "Resource Details" box appears on the right giving you various options, including deleting that resource. Unfortunately for me, that box is right below an almost identical one called "Project Details," which also includes a `Delete` button. 

<img width="1440" alt="Two delete buttons" src="https://github.com/DDMAL/Rodan/assets/83373378/4b2ec52f-be28-4319-900d-8fb11a1a71f3">

This means that a distracted user could very well delete the entire project when they only wanted to delete a resource. I have two proposed solutions for this:
- Include an "are you sure?" message when the user tries to delete a project.
- Remove the "Project Details" box when one of the tabs is selected, so that you can only see it when you select the project itself in the left-hand column.

It's very possible that I only made this mistake because I'm so used to the old rodan UI. If you think that's the case, please disregard this issue!

**Comment by JoyfulGen** [comment]
I upped the severity of this issue because it's caused me grief quite a few times now, and I believe other people have suffered my fate as well. @kyrieb-ekat had you made another issue about this?

**Comment by kyrieb-ekat** [comment]
Yes, I had made a separate issue which I closed once I realised this one existed! I have definitely deleted my project multiple times. Stefano I know also had a repeated issue with this and ultimately made an extension that just fully hid the button.

**Comment by kyrieb-ekat** [comment]
Pinging this issue again, now that it has been encountered again (see #1253; Martha also included some ideas/feedback!). I'm going to bump the priority on this and also tag Yinan. My (local) way of handling this was the Stefano method of adding a browser extension which just removed the project deletion button,  but it is definitely time for this to get addressed—will need to be a fix for staging and production. @yinanazhou, thoughts?

**Comment by yinanazhou** [comment]
Wanna follow up on this @kyrieb-ekat @JoyfulGen

**Comment by JoyfulGen** [comment]
The delete confirm window looks great on Rodan production for me! I reckon the issue can be closed, if @kyrieb-ekat agrees.
